code/modules/cargo/console.dm

```diff
@@ -17,7 +17,7 @@
 	/// var that tracks message cooldown
 	var/message_cooldown
 	var/list/loaded_coupons
-
+	var/express = FALSE
```

Comment:
Could you document this variable?
Maybe give it a better name as well?
What does `express` mean?